compiler/rustc_lint/src/invalid_from_utf8.rs

```diff
@@ -11,7 +11,7 @@ use crate::{LateContext, LateLintPass, LintContext};
 declare_lint! {
     /// The `invalid_from_utf8_unchecked` lint checks for calls to
     /// `std::str::from_utf8_unchecked` and `std::str::from_utf8_unchecked_mut`
-    /// with an invalid UTF-8 literal.
+    /// with an invalid UTF-8 literal (modulo bindings).
```

Comment:
To me this reads more like "bindings are not linted". How about "with a known invalid UTF-8 value"?

Reply:
The "module bindings" was a reference to the literal for which Waffle tough as immediate, so by adding that phrasing I was trying to imply the lint was linting on a literal but that literal might be behind some bindings.

Anyway, changed to your suggestion.